### ISSUE TITLE
Localization for Fasten

### ISSUE THREAD

**Issue by DimitriDR** [fastenhealth/fasten-onprem#36]
A localization would be great to expand accessibility to people who are not comfortable with English.

[i18n](https://www.npmjs.com/package/vue-i18n) (other variants exist) is a frequently used module in self hosted projects. Most of them use Crowdin to aggregate translations. They offer free plan for open source projects.

![image](https://user-images.githubusercontent.com/56969769/214638701-fa15bd76-2641-43f6-af88-6e56b61062eb.png)

I personally recommend it (as least as translator and proofreader).

Voilà!

I could help you, by promoting me to manager on Crowdin :)

Have a great day

**Comment by AnalogJ** [comment]
This would definitely be a great idea once we have some International provider support

related #42

**Comment by DimitriDR** [comment]
> This would definitely be a great idea once we have some International provider support
> 
> related #42

IMHO, no need for international providers. You can live in the US but having Spanish or French as first language, and therefore, It is more comfortable to have a UI in your mother tongue.

**Comment by AnalogJ** [comment]
that's a fair point.

**Comment by lardissone** [comment]
I agree with that. I don't live in the US, and for manual entering data it's a little hard to be translating studies and lab keys.

I would be happy to help with translation to Spanish.
